### PR TITLE
Issue #293 sort

### PR DESCRIPTION
**Pull request by jnorlando**

### CODE REVIEW

**Review by lil5 — CHANGES_REQUESTED** [review]
Great draggable function! Is that a in build react util or is there a `npm` dependency I'm not seeing?

File: server/internal/models/chain.go [hunk 1]
```diff
@@ -27,7 +27,8 @@ type Chain struct {
 	Sizes            []string `gorm:"serializer:json"`
 	Genders          []string `gorm:"serializer:json"`
 	UserChains       []UserChain
-	CreatedAt        time.Time
+	Route 				   []string `gorm:"serializer:json; default:[]"`
```
[the remark this answers is not in the document]
Comment (lil5) [reply server/internal/models/chain.go:30]: I know this is a little sudden, I'm currently reading **SQL Antipatterns** and this was a good example of the **Format Comma-Seperated Lists** which is an antipattern when referencing other tables. 

More information: https://github.com/boralp/sql-anti-patterns#store-each-value-in-its-own-column-and-row

I'm mostly interested in **1. Creating a column in user_chains**

File: frontend/package.json [hunk 2]
```diff
@@ -36,6 +36,7 @@
     "@types/react-helmet": "^6.1.1",
     "deck.gl": "^8.8.20",
     "events": "^3.3.0",
+    "i": "^0.3.7",
```
[the remark this answers is not in the document]
Comment (jnorlando) [reply frontend/package.json:39]: it's built in

[the remark this answers is not in the document]
Comment (jnorlando) [reply frontend/package.json:39]: draggable is built into react

**Comment by jnorlando** [comment]
Going to try to get the current revisions done between tonight and tomorrow; @lil5 Though, I'd advise not bothering to do another review of it until check-box is merged, since i'll want to sync main with the sort-fork, in order to make sure everything is still functioning as it should.

**Pull request merged**